### DISCUSSION TITLE
Kudos

### DISCUSSION THREAD

**Discussion by hiro5id** [photoprism/photoprism#2026] [General]
I would just like to say I'm impressed by this software.  It is so performant and works reasonably well on mobile phones as well.  I have tried a lot of self hosted photo management servers, but always something was missing. Either it wasn't extendable enough, or it was closed source, or too slow, or too complicated.  This seems to have the perfect balance all around.  Thanks to the maintainers and contributors. 💪

**Comment by lastzero** [comment]
Thank you, much appreciated! We do our best to make it even better 🚀
